Dockerfile.release

```diff
@@ -0,0 +1,83 @@
+ARG PY_VERSION_DEFAULT=3.9
+
+FROM debian:bullseye as base
+
+USER root
+
+ARG TARGETARCH
+ARG MG_VERSION=2.1.1
+ARG PY_VERSION_DEFAULT
+ENV MG_VERSION ${MG_VERSION}
+ENV PY_VERSION ${PY_VERSION_DEFAULT}
+
+#essentials for production/dev
+RUN apt-get update && apt-get install -y \
+    libcurl4        `memgraph` \
+    libpython${PY_VERSION}   `memgraph` \
+    libssl1.1       `memgraph` \
+    openssl         `memgraph` \
+    build-essential `mage-memgraph` \
+    cmake           `mage-memgraph` \
+    curl            `mage-memgraph` \
+    g++             `mage-memgraph` \
+    python3         `mage-memgraph` \
+    python3-pip     `mage-memgraph` \
+    python3-setuptools     `mage-memgraph` \
+    python3-dev     `mage-memgraph` \
+    uuid-dev        `mage-memgraph` \
+    clang           `mage-memgraph` \
+    git             `mage-memgraph` \
+    --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY memgraph-${TARGETARCH}.deb .
```

Comment:
To prevent this, I think we should aim for better naming on our download bucket :joy:
There is much redundant stuff here now, although each of them seems to be necessary for having continuous integration.

Reply:
That's true we will have discussion on naming packages for different distros